vendor/knative.dev/pkg/controller/stats_reporter.go

```diff
@@ -199,7 +199,7 @@ func (r *reporter) ReportReconcile(duration time.Duration, success string, key t
 		return err
 	}
 
-	// TODO skonto: fix latency histogram
-	metrics.RecordBatch(ctx, reconcileCountStat.M(1))
+	metrics.RecordBatch(ctx, reconcileCountStat.M(1),
+		reconcileLatencyStat.M(duration.Milliseconds()))
```

Comment:
/hold

This is reverting this patch https://github.com/openshift/knative-eventing/blob/release-v1.4/openshift/patches/020-mutemetrics.patch, please apply git patches with `git apply openshift/patcher/*`